dynawo/sources/Modeler/DataInterface/PowSyblIIDM/test/TestVoltageLevelInterface.cpp

```diff
@@ -166,6 +166,201 @@ TEST(DataInterfaceTest, VoltageLevel) {
                           .setHighVoltageLimit(420.)
                           .setLowVoltageLimit(380.)
                           .add();
+//   vlIIDM2.getNodeBreakerView().newBusbarSection()
```

Comment:
why this part is commented?

Reply:
Relic from developement. will be removed